reference/7/Microsoft.PowerShell.Core/About/about_Pipeline_Chain_Operators.md

```diff
@@ -14,27 +14,32 @@ Describes chaining pipelines with the `&&` and `||` operators in PowerShell.
 
 ## Long description
 
-From PowerShell 7, PowerShell implements the `&&` and `||` operators
-to conditionally chain pipelines.
-These operators are known in PowerShell as *pipeline chain operators*,
-and are similar to [AND-OR lists](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_09_03)
-in POSIX shells like bash, zsh and sh,
-as well as [conditional processing symbols](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb490954(v=technet.10)#using-multiple-commands-and-conditional-processing-symbols)
-in Windows' command shell (cmd.exe).
-
-`&&` and `||` allow conditional execution of PowerShell commands and pipelines
-based on the success of the left-hand pipeline.
-In particular, they allow conditional execution of native executables
-based on execution success:
+Beginning in PowerShell 7, PowerShell implements the `&&` and `||` operators to
+conditionally chain pipelines. These operators are known in PowerShell as
+*pipeline chain operators*, and are similar to
+[AND-OR lists](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_09_03)
+in POSIX shells like bash, zsh and sh, as well as
+[conditional processing symbols](/previous-versions/windows/it-pro/windows-xp/bb490954(v=technet.10)#using-multiple-commands-and-conditional-processing-symbols)
+in the Windows Command Shell (cmd.exe).
+
+The `&&` operator executes the right-hand pipeline, if the left-hand pipeline
+succeeded. Conversely, the `||` operator executes the right-hand pipeline if
+the left-hand pipeline failed.
+
+This can be use for conditional execution of software deployments. For example:
```

Comment:
`use` -> `used`.

The language here feels too specific for an operator to me.

If this example demands language like that, we should change the example.

I want to be able to emphasise that this works with native commands based on `$?`/`$LASTEXITCODE` and not just with cmdlets/functions.